### PR TITLE
Disable openmp when raja is not used

### PR DESCRIPTION
**Pull request by nychiang**
fix https://github.com/xsdk-project/xsdk-issues/issues/181

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -181,7 +181,7 @@ if(HIOP_DEVELOPER_MODE)
   install(TARGETS hiop_options hiop_warnings EXPORT hiop-targets)
 endif()
 
-if(NOT (HIOP_USE_GPU AND HIOP_USE_HIP))
+if(HIOP_USE_RAJA AND (NOT (HIOP_USE_GPU AND HIOP_USE_HIP)))
```
Comment (CameronRutherford) [inline CMakeLists.txt:184]: Agreed that this closes the xSDK issue, however, we should update in future to fix this based on our discussion.

**Review by CameronRutherford — APPROVED** [review]

**Comment by CameronRutherford** [comment]
It would be great if we could tag another minor release to have xSDK builds passing on mac

**Comment by cnpetra** [comment]
> It would be great if we could tag another minor release to have xSDK builds passing on mac

will prepare tag 0.7.1 sometime before Monday.

File: CMakeLists.txt (hunk 1 above)
Comment (nychiang) [reply CMakeLists.txt:184]: I agree! I saw you have already created an issue for it (#563)

**Pull request merged**